crates/sui-bridge/src/test_utils.rs

```diff
@@ -231,3 +245,192 @@ pub fn get_test_log_and_action(
     });
     (log, bridge_action)
 }
+
+pub async fn publish_bridge_package(context: &WalletContext) -> BTreeMap<TokenId, ObjectRef> {
+    let (sender, gas_object) = context.get_one_gas_object().await.unwrap().unwrap();
+    let gas_price = context.get_reference_gas_price().await.unwrap();
+
+    let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    path.extend(["..", "..", "examples", "move", "bridge"]);
+
+    let txn = context.sign_transaction(
+        &TestTransactionBuilder::new(sender, gas_object, gas_price)
+            .publish(path)
+            .build(),
+    );
+    let resp = context.execute_transaction_must_succeed(txn).await;
+    let object_changes = resp.object_changes.unwrap();
+    let package_id = object_changes
+        .iter()
+        .find(|change| matches!(change, ObjectChange::Published { .. }))
+        .map(|change| change.object_id())
+        .unwrap();
+
+    let mut treasury_caps = BTreeMap::new();
+    object_changes.iter().for_each(|change| {
+        if let ObjectChange::Created { object_type, .. } = change {
+            let object_type_str = object_type.to_string();
+            if object_type_str.contains("TreasuryCap") {
+                if object_type_str.contains("BTC") {
```

Comment:
Not sure if similar logic exists in the production version. If it is, it probably should match exact tickers.

Reply:
what does "exact tickers" mean?

Reply:
got it. No I don't think we will have exact same logic in production

Reply:
Yeah just in case anyone can create an `object_type_str` containing `BTC` and cause confusions.